File: identicon/src/main/java/com/lelloman/identicon/util/TileMeasures.java
package com.lelloman.identicon.util;

/**
 * 	Utility class that holds some useful points for drawing all the shapes
 */
public class TileMeasures {

	public final int width,height,wMid,hMid,w3,h3,w4,h4,w32,h32,w43,h43,width2,height2, width3, height3;

	public TileMeasures(int width,int height){
		this.width = width;
		this.height = height;

		wMid = width / 2;
		hMid = height / 2;
		w3 = width / 3;
		h3 = height / 3;
		w4 = width / 4;
		h4 = height / 4;
		w32 = w3 * 2;
		h32 = h3 * 2;
		w43 = w4 * 3;
		h43 = h4 * 3;

		width2 = width * 2;
		height2 = height * 2;

		width3 = width * 3;
		height3 = height * 3;
	}
}

File: identicon/src/main/java/com/lelloman/identicon/util/TileMeasures.kt
package com.lelloman.identicon.util

/**
 * Utility class that holds some useful points for drawing all the shapes.
 */
data class TileMeasures(val width: Int, val height: Int) {
    val wMid: Int = width / 2
    val hMid: Int = height / 2
    val w3: Int = width / 3
    val h3: Int = height / 3
    val w4: Int = width / 4
    val h4: Int = height / 4
    val w32: Int = w3 * 2
    val h32: Int = h3 * 2
    val w43: Int = w4 * 3
    val h43: Int = h4 * 3
    val width2: Int = width * 2
    val height2: Int = height * 2
}
